### DISCUSSION TITLE
width on media queries not working

### DISCUSSION THREAD

**Discussion by titocs** [tailwindlabs/tailwindcss#8540] [Help] [answered]
i want to make social media logos width responsive, in screen with large < 768 i set the width to w-5 and w-4, and for 768 and larger i set the width to w-10, but when i change the size of screen to 768, i've tried to restart terminal but still no change. what should i do?
image 1 (screen < 768) : https://ibb.co/4dxQ3vs
image 2 (screen >= 768() : https://ibb.co/PmFbyCn

**Comment by carlos-dubon** [comment, marked as answer]
try with:
`class="w-5 !md:w-10"`
